content/learn/about.md

```diff
@@ -0,0 +1,56 @@
+---
+menu: learn
```

Comment:
Looks like the https://github.com/hybrid-cloud-patterns/docs/blob/gh-pages/workflow.md has not been migrated under the `learn` directory as listed in the _Jekyll to Hugo listing_ tracking sheet. Although a comment corresponding to the file in the sheet mentions that some of the file contents could be moved to the `contribute` directory, I feel we should still migrate the file under `learn` for now.

Reply:
So I moved it manually because I thought the sub-pages seemed more in-line with contribute rather than learn. That said, I'm open to moving it (and the subpages) back to learn